index.html

```diff
@@ -1,13 +1,183 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="assets/css/styles.css">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="assets/css/styles.css">
+  <!-- Cocogoose font family -->
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <!-- Lato font family -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
+  <script>
+
+  </script>
+  <title>Annual SMSE Summit welcome page</title>
 </head>
 <body>
+  <header>
+    <!--ADD THE FIRST LEVEL -->
+    <div class="container-fluid my-display-desktop social" id="social">
+      
+    </div>
+    <!--ADD THE NAVBAR -->
+    <nav class="navbar navbar-expand-md navbar-light bg-light sticky-top" id="my-navbar">
+      
+    </nav>
```

Comment:
- [OPTIONAL] Although this requirement is missed by the previous reviewer, I would highly recommend that you implement it. Kindly have a look at your design and the original design. 
In Original design in both Home and About pages, the hamburger menu has a transparent background and there is no nav background there. Neither the upper-level nav nor the lower level nav, so kindly consider removing these two navs in the mobile screen in both home and about pages.
And also consider to add Home page link in the navlinks in mobile screen for easy browsing.
your design
![fd](https://user-images.githubusercontent.com/96683843/235319882-0efc8a97-8980-4a52-ab5e-ab1edc149b92.jpg)

Original design

![as](https://user-images.githubusercontent.com/96683843/235319877-febfe775-b433-4767-9d95-e134c4f0cef6.jpg)

Reply:
The Section has been removed for all 2 pages, thank you for the comment. @zouhaib
I will work towards fixing the issue.

![image](https://user-images.githubusercontent.com/36234985/235321216-c5a66522-ce5a-446b-905a-145d6ff4c82f.png)


![image](https://user-images.githubusercontent.com/36234985/235321245-22e3fe9a-5f3e-4cd2-830c-464192df924d.png)